### PR TITLE
Add listen new config param "minimum_level" error log.

### PR DESCRIPTION
**Pull request by Yozhef**
Hello everybody.

Problem:
In the project, I am currently working on it. There is a createRollbarHandler function in the RollbarHandlerFactory class. This LogLevel parameter accepts only Error / Critical values. It would be great if it is possible to put it. Because info info includes `Warning / Warning / Error / Critical`.

Solution for the problem:
I created you PR on [rollbar-php](https://github.com/rollbar/rollbar-php/pull/440) when i add new param in config `minimum_level`, and this i use him and determine which error level is required.

Example config:
```
rollbar:
    minimum_level: 'warning'
```

### CODE REVIEW

File: Factories/RollbarHandlerFactory.php [hunk 1]
```diff
@@ -30,30 +35,26 @@ public function __construct(ContainerInterface $container)
...
-                
-            }
-            
+                } catch (\Exception $exception) {
+                    // Ignore
+                }
+            };
         }
 
+        $this->minimumLevel = $config['minimum_level'] ?: LogLevel::ERROR;
```
Comment (ArturMoczulski) [inline Factories/RollbarHandlerFactory.php]: Same here, I recommend using `monolog_minimum_level` for the config option name as suggested in https://github.com/rollbar/rollbar-php/pull/440#pullrequestreview-215372893

Also, I think `isset($config['minimum_level'])` is needed here as I get PHP notices if `minimum_level` is not set in `rollbar.yaml`:

```php
$this->minimumLevel = isset($config['minimum_level']) ? $config['minimum_level'] : LogLevel::ERROR;
```

Comment (Yozhef) [reply Factories/RollbarHandlerFactory.php]: @ArturMoczulski okay, i change it.

**Comment by Yozhef** [comment]
@ArturMoczulski I added as you requested, but I think verification is not required because we add  `monolog_minimum_level` -> default valuer `LogLevel::ERROR`. 

I think if we add to (default, config) value need on code only get.
1) If a person does not use in his config yaml this params -> аutomatically uses value from  default.
2) If a person use on his config this param, he override default values.

**Comment by ArturMoczulski** [comment]
Let's continue this work in https://github.com/rollbar/rollbar-php-symfony-bundle/pull/46

**Pull request closed without merging**